gems/sorbet/lib/fetch-rbis.rb

```diff
@@ -14,12 +14,12 @@ class Sorbet::Private::FetchRBIs
   SORBET_DIR = 'sorbet'
   SORBET_CONFIG_FILE = "#{SORBET_DIR}/config"
   SORBET_RBI_LIST = "#{SORBET_DIR}/rbi_list"
-  SORBET_RBI_SORBET_TYPED = "#{SORBET_DIR}/rbi/sorbet-typed/"
+  SORBET_RBI_SORBET_TYPED = "#{SORBET_DIR}/rbi/sorbet-typed"
 
   XDG_CACHE_HOME = ENV['XDG_CACHE_HOME'] || "#{ENV['HOME']}/.cache"
   RBI_CACHE_DIR = "#{XDG_CACHE_HOME}/sorbet/sorbet-typed"
 
-  SORBET_TYPED_REPO = 'https://github.com/sorbet/sorbet-typed.git'
+  SORBET_TYPED_REPO = ENV['SRB_SORBET_TYPED_REPO'] || 'https://github.com/sorbet/sorbet-typed.git'
   SORBET_TYPED_REVISION = ENV['SRB_SORBET_TYPED_REVISION'] || 'origin/master'
```

Comment:
Our original intention of this `SRB_SORBET_TYPED_REVISION` environment variable was to do exactly what it seems you're trying to do in this PR. e.g.,

- cd into the cache dir
- git remote add <the fork>
- export SRB_SORBET_TYPED_REVISION="<the PR branch name>"

In general, we'd prefer to not have more than one way to do things. Given this:

> where previously we had to go into ~/.cache/sorbet/sorbet-typed/ and mess with the git repo in there

it seems to me like mucking with an environment variable is already "messing with `srb` internals," so going into `~/.cache` seems like it shouldn't be a huge step. And it should always be safe to just nuke that directory if it gets into a weird git state.

Reply:
I disagree that mucking with the git repo in `~/.cache/` is as easy as setting an environment variable, e.g. it's the difference between being able to just run `SRB_SORBET_TYPED_REPO=https://github.com/connorshea/sorbet-typed.git bundle exec srb rbi sorbet-typed` vs. needing to:
- `cd ~/.cache/sorbet/sorbet-typed/`
- `git remote add fork https://github.com/connorshea/sorbet-typed.git`
- `SRB_SORBET_TYPED_REVISION=fork/master bundle exec srb rbi sorbet-typed`

It does become essentially equal after the first initial setup, but I'd still prefer to be able to just use `SRB_SORBET_TYPED_REPO=https://github.com/connorshea/sorbet-typed.git bundle exec srb rbi sorbet-typed` whenever I want to pull from a specific remote.

Admittedly, I didn't know about the REVISION environment variable, and didn't really think of being able to add a git repo as a remote rather than overriding the `origin`.

I guess I'd be fine with adding the environment variable for specifying the remote, though I still think this is a simpler solution in the end.

Also, since the code below does `git fetch origin` before `git checkout -q SORBET_TYPED_REVISION`, I'm not actually sure if the checkout would work for a remote other than `origin` (unless you went into the cache directory and fetched). 🤔 I'd need to test that.